src/landing-page/types.ts

```diff
@@ -1,28 +0,0 @@
-import { ViewerChannelSettings } from "../aics-image-viewer/shared/utils/viewerChannelSettings";
-
-// TBD what URL parameters to include here
-export type ViewerArgs = {
-  baseurl: string;
-  cellid: number;
-  cellPath: string;
-  fovPath: string;
-  fovDownloadHref: string;
-  cellDownloadHref: string;
-  viewerSettings: ViewerChannelSettings;
-};
```

Comment:
This was removed and replaced with the `AppProps` type that Cameron introduced in another PR.